webpack.config.js

```diff
@@ -246,10 +246,6 @@ if ( calypsoEnv === 'desktop' ) {
 	// no chunks or dll here, just one big file for the desktop app
 	webpackConfig.output.filename = '[name].js';
 	webpackConfig.output.chunkFilename = '[name].js';
-} else {
-	// jquery is only needed in the build for the desktop app
-	// see electron bug: https://github.com/atom/electron/issues/254
-	webpackConfig.externals.push( 'jquery' );
```

Comment:
Ah we'll need to make sure we remove this before change before merge. This is still used by Desktop

Reply:
Is it still there? I hoped you would refactor `paygate` to be `jQuery` free :D

Reply:
@astralbodies did anything change with regards to the jQuery include? Pretty sure we still need this, right?

Reply:
it's required by [`@wordpress/api-fetch`](https://github.com/WordPress/gutenberg/blob/1a0b22a6bad45dc001e56f6a199521db0a7bc29f/packages/api-fetch/package.json#L25), which is a [dependency of `@wordpress/components`](https://github.com/WordPress/gutenberg/blob/1a0b22a6bad45dc001e56f6a199521db0a7bc29f/packages/components/package.json#L24)

Reply:
> it's required by @wordpress/api-fetch, which is a dependency of @wordpress/components

I think we should get rid of the jQuery dependency here. The issue in this case is that jQuery in core triggers events on its own events layer, which we can't listen to with native `addEventListener`s. @gziolo had a good idea: use `wp.hooks` to relay the jQuery event into a native event, which can then be listened to by `api-fetch`. Possibly by adding some code to do this as an inline script: https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L154-L161

I think it would also be a good idea to audit what we're depending on jQuery for in Gutenberg in general, and see what can be factored out.

Reply:
Thanks for the suggestion! I have applied it in WordPress/gutenberg#8311, so as soon as that is merged and published to npm I'll remove the `jQuery` dependency.

Reply:
@ehg I have reverted the change and jQuery is not needed now in the bundle. Is this ready to go now?